packages/extension/src/components/DaHeader.vue

```diff
@@ -281,16 +281,18 @@ export default {
   }
 
   & .header__top-site {
-    width: 24px;
-    height: 24px;
-    border-radius: 50%;
+    width: 28px;
+    height: 28px;
+    border-radius: 4px;
```

Comment:
Let's change the border-radius to be `8px`.
It fits better our style guide.

```suggestion
    border-radius: 8px;
```